qt/scientific_interfaces/ISISReflectometry/Common/Detector.cpp

```diff
@@ -0,0 +1,27 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2022 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "Detector.h"
+
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidGeometry/Instrument.h"
+
+namespace MantidQt {
```

Comment:
This could use short notation for namespaces i.e. `namespace MantidQt::CustomInterfaces::ISISReflectometry`